src/coreclr/nativeaot/Runtime/loongarch64/ExceptionHandling.S

```diff
@@ -0,0 +1,831 @@
+        ld.d  $a3, $a2, OFFSETOF__Thread__m_pExInfoStackHead
+        st.d  $a3, $a1, OFFSETOF__ExInfo__m_pPrevExInfo       // pExInfo->m_pPrevExInfo = m_pExInfoStackHead
+        st.d  $a1, $a2, OFFSETOF__Thread__m_pExInfoStackHead  // m_pExInfoStackHead = pExInfo
+
+        // set the exception context field on the ExInfo
+        addi.d  $a2, $sp, rsp_offsetof_Context                // a2 <- PAL_LIMITED_CONTEXT*
+        st.d  $a2, $a1, OFFSETOF__ExInfo__m_pExContext        // pExInfo->m_pExContext = pContext
+
+        // a0: exception object
+        // a1: ExInfo*
+        bl  C_FUNC(RhThrowEx)
+
+    ALTERNATE_ENTRY RhpThrowEx2
+
+        // no return
+        EMIT_BREAKPOINT
+    NESTED_END RhpThrowEx, _TEXT
+
+
+//
+// void FASTCALL RhpRethrow()
+//
+// SUMMARY:  Similar to RhpThrowEx, except that it passes along the currently active ExInfo
+//
+// INPUT:
+//
+// OUTPUT:
+//
+
+    NESTED_ENTRY RhpRethrow, _TEXT, NoHandler
+
+        ALLOC_THROW_FRAME SOFTWARE_EXCEPTION
+
+        GetThreadX2
+
+        addi.d  $a1, $sp, rsp_offsetof_ExInfo            // a1 <- ExInfo*
+        st.d  $zero, $a1, OFFSETOF__ExInfo__m_exception  // pExInfo->m_exception = null
+        st.b  $zero, $a1, OFFSETOF__ExInfo__m_kind       // init to a deterministic value (ExKind.None)
+        ori  $a3, $zero, 1
+        st.b  $a3, $a1, OFFSETOF__ExInfo__m_passNumber   // pExInfo->m_passNumber = 1
+        lu12i.w  $a3, -1
+        ori  $a3, $a3, 0xfff
+        st.w  $a3, $a1, OFFSETOF__ExInfo__m_idxCurClause // pExInfo->m_idxCurClause = MaxTryRegionIdx
+
+        // link the ExInfo into the thread's ExInfo chain
+        ld.d  $a3, $a2, OFFSETOF__Thread__m_pExInfoStackHead
+        ori  $a0, $a3, 0                                      // a0 <- current ExInfo
+        st.d  $a3, $a1, OFFSETOF__ExInfo__m_pPrevExInfo       // pExInfo->m_pPrevExInfo = m_pExInfoStackHead
+        st.d  $a1, $a2, OFFSETOF__Thread__m_pExInfoStackHead  // m_pExInfoStackHead = pExInfo
+
+        // set the exception context field on the ExInfo
+        addi.d  $a2, $sp, rsp_offsetof_Context                // a2 <- PAL_LIMITED_CONTEXT*
+        st.d  $a2, $a1, OFFSETOF__ExInfo__m_pExContext        // pExInfo->m_pExContext = pContext
+
+        // a0 contains the currently active ExInfo
+        // a1 contains the address of the new ExInfo
+        bl  C_FUNC(RhRethrow)
+
+    ALTERNATE_ENTRY RhpRethrow2
+
+        // no return
+        EMIT_BREAKPOINT
+    NESTED_END RhpRethrow, _TEXT
+
+//
+// void* FASTCALL RhpCallCatchFunclet(OBJECTREF exceptionObj, void* pHandlerIP, REGDISPLAY* pRegDisplay,
+//                                    ExInfo* pExInfo)
+//
+// INPUT:  a0:  exception object
+//         a1:  handler funclet address
+//         a2:  REGDISPLAY*
+//         a3:  ExInfo*
+//
+// OUTPUT:
+//
+
+    NESTED_ENTRY RhpCallCatchFunclet, _TEXT, NoHandler
+
+        ALLOC_CALL_FUNCLET_FRAME 0x70 // Size needs to be equal with ExceptionHandling.asm variant of this function
```

Comment:
```suggestion
        ALLOC_CALL_FUNCLET_FRAME 0x70
```

Reply:
OK, I've updated these in my latest commit.
Thanks!